python/tvm/relay/expr.py

```diff
@@ -27,6 +27,13 @@ def __call__(self, *args):
         return Call(self, args, None, None)
 
 
+    def alpha_eq(self, other):
+        """Compare two Relay expr for structural equivalence using
```

Comment:
document the parameters, as in https://docs.tvm.ai/contribute/document.html#document-python